src/components/Screens/HomeScreen.tsx

```diff
@@ -244,6 +245,72 @@ function InsightBudgets() {
   );
 }
 
+function Bills() {
+  const { colors } = useThemeColors();
+
+  const billsSummary = useSelector((state: RootState) => state.firefly.bills);
+  const bills = useSelector((state: RootState) => state.bills.bills);
+  const loading = useSelector((state: RootState) => state.loading.effects.bills?.getBills?.loading);
+  const dispatch = useDispatch<RootDispatch>();
+
+  const totalPaid = useMemo(() => parseFloat(billsSummary?.paid?.monetaryValue || '0'), [billsSummary]);
+  const totalUnpaid = useMemo(() => Math.abs(parseFloat(billsSummary?.unpaid?.monetaryValue || '0')), [billsSummary]);
+  const total = useMemo(() => totalPaid + totalUnpaid, [totalPaid, totalUnpaid]);
+
+  return (
+    <AScrollView
+      showsVerticalScrollIndicator={false}
+      refreshControl={(
+        <RefreshControl
+          refreshing={false}
+          onRefresh={() => dispatch.bills.getBills()}
+        />
+      )}
+    >
+      <AStack row>
+        <AText fontSize={25} lineHeight={27} style={{ margin: 15, flex: 1 }} bold>
+          {translate('home_bills')}
+        </AText>
+
+        <AStack
+          px={6}
+          py={2}
+          mx={15}
+          backgroundColor={colors.brandSuccessLight}
+          style={{ borderRadius: 5 }}
+        >
+          <AText
+            fontSize={15}
+            numberOfLines={1}
+            color={colors.brandSuccess}
+            style={{ textAlign: 'center' }}
+            bold
+          >
+            {`${((totalPaid / total) * 100).toFixed(0)}%`}
+          </AText>
+        </AStack>
+      </AStack>
+
+      <AStack mx={15} justifyContent="flex-start">
+        <AProgressBar
+          color={colors.green}
+          value={(totalPaid / total) * 100}
+        />
+      </AStack>
```

Comment:
(same comment) Can you wrap this stack to conditionally display it if only `total` !== 0?

For older periods where I have bills but nothing is expected total return 0, displaying `NaN` in the "bills summary".